src/components/base.scss

```diff
@@ -5,8 +5,6 @@
 .be {
     @include reset;
 
-    @import '~box-react-ui/lib/styles/reset';
```

Comment:
Hmm, wasn't this already resetting scoped to .be?

Reply:
Via this import, yes, but it wasn't playing well with Details tab. Appears to be the only thing activity feed was using was the `ol, ul` style added to ActivityFeed.scss